src/main/java/edu/harvard/h2ms/domain/core/User.java

```diff
@@ -49,13 +36,12 @@
 
   @NotNull @Column private String type;
 
-  @NotNull @Column @JsonIgnore private String password;
+  @NotNull @Column private String password;

```

Comment:
`@JsonIgnore` prevented the hashed password being exposed in API responses, maybe the wrong annotation and I think  we need `@JsonProperty(access = Access.WRITE_ONLY)` [instead](https://stackoverflow.com/questions/12505141/only-using-jsonignore-during-serialization-but-not-deserialization)

Reply:
@shinwachi @kevin -- I think one of guys may have already handled this.

Reply:
As far as I can tell it's not, you're removing JsonIgnore in this PR which will make the passwords show up in the API responses.  I think you can add it to the getter or my original suggestion of JsonProperty